gammapy/estimators/flux.py

```diff
@@ -68,12 +85,23 @@ def __init__(
         selection_optional=None,
         fit=None,
         reoptimize=False,
+        norm=None,
```

Comment:
I was just wondering whether we should make the name a little bit more clearer, to avoid the confusion of an estimator having a `Parameter` object. For example `norm_spec` or similar.

Reply:
The existing docstring already explains that a "norm" parameter is used to rescale the reference model so it should not be that surprising for the user :
`The amplitude is re-normalized using the "norm" parameter,
    which specifies the deviation of the flux from the reference model in this
    energy range.`
 and I would prefer to keep it short so it is easier to set the attributes.

Reply:
added more details in the docsting